.github/workflows/ci.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         with:
           repository: NREL/resstock
           path: resstock
-          ref: develop
+          ref: res-reporting
```

Comment:
Before merge, need to switch this and down below back to develop (and make sure tests are still passing).